CHANGELOG.md

```diff
@@ -1,5 +1,7 @@
 # Change Log
 
+## 0.3.2 (February 9th, 2021)
+- enhancement - Usage data collection to enhance extension. For more details view [privacy statement](https://developers.redhat.com/article/tool-data-collection) and [usage data doc](Telemetry.md). See [#489](https://github.com/fabric8-analytics/fabric8-analytics-vscode-extension/pull/486) [#487](https://github.com/fabric8-analytics/fabric8-analytics-vscode-extension/pull/487) [#488](https://github.com/fabric8-analytics/fabric8-analytics-vscode-extension/pull/488)
```

Comment:
```suggestion
- enhancement - Usage data collection to enhance extension. For more details view [privacy statement](https://developers.redhat.com/article/tool-data-collection) and [usage data doc](Telemetry.md). See [#489](https://github.com/fabric8-analytics/fabric8-analytics-vscode-extension/pull/486) [#487](https://github.com/fabric8-analytics/fabric8-analytics-vscode-extension/pull/487) [#488](https://github.com/fabric8-analytics/fabric8-analytics-vscode-extension/pull/488)

```

You can mention this too https://github.com/fabric8-analytics/fabric8-analytics-vscode-extension/pull/485

Reply:
done

Reply:
You may need to add https://github.com/fabric8-analytics/fabric8-analytics-vscode-extension/pull/494

Reply:
ok